deploy/template.yaml

```diff
@@ -1087,6 +1087,9 @@ Resources:
         IntegrationResponses:
           - StatusCode: "200"
             SelectionPattern: "200"
+            ResponseParameters:
+              integration.response.header.Content-Type: "application/did+json"
```

Comment:
Think this should be `method.` rather than `integration.`
```suggestion
              method.response.header.Content-Type: "application/did+json"
```